Log levels higher than "V(1)" cannot be enabled

### Summary

When I enabled `--log-level=debug`, I was expecting to see log output for call sites that use `log.V(2).Info(...)`, but nothing appears.

This patch demonstrated the problem:
```Go
-- app/kuma-cp/cmd/run.go
+++ app/kuma-cp/cmd/run.go
@@ -93,6 +93,10 @@ func newRunCmdWithOpts(opts runCmdOpts) *cobra.Command {
                                        "minimim-open-files", minOpenFileLimit)
                        }
 
+                       if !runLog.V(2).Enabled() {
+                               panic("V(2) not enabled")
+                       }
+
                        switch cfg.Mode {
                        case config_core.Standalone:
                                if err := mads_server.SetupServer(rt); err != nil {
```
```
$ kuma-cp
2021-07-12T15:38:19.115+1000	INFO	Skipping reading config from file
2021-07-12T15:38:19.115+1000	INFO	bootstrap.auto-configure	directory /home/jpeach/.kuma will be used as a working directory, it could be changed using KUMA_GENERAL_WORK_DIR environment variable
...
2021-07-12T15:38:19.117+1000	INFO	kuma-cp.run	Running in mode `standalone`
panic: V(2) not enabled

goroutine 1 [running]:
github.com/kumahq/kuma/app/kuma-cp/cmd.newRunCmdWithOpts.func1(0xc000baef00, 0xc0003d3390, 0x0, 0x1, 0x0, 0x0)
	/home/jpeach/upstream/konghq/kuma/app/kuma-cp/cmd/run.go:97 +0x1be5
github.com/spf13/cobra.(*Command).execute(0xc000baef00, 0xc0003d3380, 0x1, 0x1, 0xc000baef00, 0xc0003d3380)
	/home/jpeach/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc000baec80, 0x41af01, 0x0, 0x0)
	/home/jpeach/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/jpeach/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/kumahq/kuma/app/kuma-cp/cmd.Execute()
	/home/jpeach/upstream/konghq/kuma/app/kuma-cp/cmd/root.go:73 +0x2a
main.main()
	/home/jpeach/upstream/konghq/kuma/app/kuma-cp/main.go:6 +0x25

```

--- a/pkg/log/logger.go
+++ b/pkg/log/logger.go
@@ -69,7 +69,11 @@ func newZapLoggerTo(destWriter io.Writer, level LogLevel, opts ...zap.Option) *z
 	case OffLevel:
 		return zap.NewNop()
 	case DebugLevel:
-		lvl = zap.NewAtomicLevelAt(zap.DebugLevel)
+		// The value we pass here is the most verbose level that
+		// will end up being emitted through the `V(level int)`
+		// accessor. Passing -10 ensures that levels up to `V(10)`
+		// will work, which seems like plenty.
+		lvl = zap.NewAtomicLevelAt(-10)
 		opts = append(opts, zap.Development(), zap.AddStacktrace(zap.ErrorLevel))
 	default:
 		lvl = zap.NewAtomicLevelAt(zap.InfoLevel)

